src/types/request.ts

```diff
@@ -23,6 +23,7 @@ export interface GetSchedulesRequestQuery {
   group:     Group['id'];
   dateStart: string;
   dateEnd:   string;
+  subgroups: string | null;
```

Comment:
Если параметр необязателен, то 
```suggestion
  subgroups?: string;
```

Знак ? перед : будет тем же самым что и
```
  subgroups: string | undefined;
```